crates/uv/tests/pip_install_scenarios.rs

```diff
@@ -1507,28 +1507,24 @@ fn local_transitive_greater_than() {
         .arg("local-transitive-greater-than-a")
                 .arg("local-transitive-greater-than-b==2.0.0+foo")
         , @r###"
-    success: true
-    exit_code: 0
+    success: false
+    exit_code: 1
     ----- stdout -----
 
     ----- stderr -----
-    Resolved 2 packages in [TIME]
-    Downloaded 2 packages in [TIME]
-    Installed 2 packages in [TIME]
-     + package-a==1.0.0
-     + package-b==2.0.0+foo
+      × No solution found when resolving dependencies:
+      ╰─▶ Because only package-a==1.0.0 is available and package-a==1.0.0 depends on package-b>2.0.0, we can conclude that all versions of package-a depend on package-b>2.0.0.
+          And because you require package-a and you require package-b==2.0.0+foo, we can conclude that the requirements are unsatisfiable.
```

Comment:
Not for here, but I wonder if we should add a "hint" explaining why `package-b==2.0.0+foo` does not satisfy `package-b>2.0.0`